classification/src/main/java/dev/ikm/komet/reasoner/ReasonerResultsNode.java

```diff
@@ -174,15 +194,15 @@ private void fullReasoner() {
 		});
 	}
 
-
-	private void elkOwlReasonerIncremental(ActionEvent actionEvent) {
-		reinferAllHierarchy = false;
+	private void runIncrementalReasoner() {
+		if (!confirmRun("incremental"))
```

Comment:
Can we make literal Strings as constants? magic values: `"incremental"`?

Reply:
Don't know what magic values are

Reply:
I meant magic numbers. You might consider making an `enum` or constant string.

```java
  enum ReasonerRunState {
      INCREMENTAL,
      FULL
  }
// or
 public static final String INCREMENTAL = "incremental";



...// later in code
if (!confirmRun(INCREMENTAL)) {
   // stuff
}
``` 

https://stackoverflow.com/questions/47882/what-are-magic-numbers-and-why-do-some-consider-them-bad